Another projekts/calculator/main_calculator_1.py

```diff
@@ -0,0 +1,37 @@
+run = True
+
+while run:
+    what_todo = input("What do you want to do?\n1 for +\n2 for -\n3 for *\n4 for /\n: ")
+
+    def mathematical_operation():
+        first_number = int(input("Enter first number: "))
+        second_number = int(input("Enter second number: "))
+        if what_todo == '1':
+            result = first_number + second_number
+        elif what_todo == '2':
+            result = first_number - second_number
+        elif what_todo == '3':
+            result = first_number * second_number
+        elif what_todo == '4':
+            result = first_number / second_number
+        print(f"Answer is {result}")
+
+    match what_todo:
+        case '1':
+            mathematical_operation()
+        case '2':
+            mathematical_operation()
+        case '3':
+            mathematical_operation()
+        case '4':
+            mathematical_operation()
```

Comment:
You can add a parameter to `mathematical_operation()` like this:
```
mathematical_operation(todo_stage)
```

So you can avoid `match-case` duplication.
Can you see the duplication? 
`case '1':` is equal to `if what_todo == '1':`
Basically, you need first check if there is no `_` matching and then execute `mathematical_operation()` in a `while run:` loop.

Reply:
> You can add a parameter to `mathematical_operation()` like this:
> 
> ```
> def mathematical_operation(operation):
> ```
> 
> This is a good code style. Better than using the global variables. This makes your function portable to the other code. And you can avoid `match-case` duplication as well. Can you see the duplication? `case '1':` is equal to `if what_todo == '1':` Basically, you need first check if there is no `_` matching and then execute `mathematical_operation(what_todo)` in a `while run:` loop.

Test answer

Reply:
test answer 1

Reply:
This is the way to go except for the function definition.
I hope you can refactor more this code.